oled.js

```diff
@@ -39,7 +39,7 @@ var Oled = function(i2c, opts) {
   this.cursor_y = 0;
 
   // new blank buffer
-  this.buffer = new Buffer((this.WIDTH * this.HEIGHT) / 8);
+  this.buffer = Buffer.alloc((this.WIDTH * this.HEIGHT) / 8);
```

Comment:
`Buffer.alloc` has appeared in version 6.0.0. It makes this lib incompatible with older version of Node.js. Can we make check if `Buffer.alloc` exists and use `new Buffer` if not exists?